apps/web/src/containers/LandingPageView/components/our-products/hooks/use-provide-liquidity-max-rate.tsx

```diff
@@ -0,0 +1,31 @@
+import { useCurrency } from '@notional-finance/notionable-hooks';
+import { NTokenValue } from '@notional-finance/sdk/src/system';
+import { Market } from '@notional-finance/sdk/src/system';
+
+interface MaxRateProvideLiquidityData {
+  symbol: string;
+  maxRate: string;
+}
+
+export const useProvideLiquidityMaxRate = (): MaxRateProvideLiquidityData => {
+  const { tradableCurrencies } = useCurrency();
+  let currentRate = 0;
+  let maxRateProvideLiquidityData = {
```

Comment:
Similar comment to lend borrow rates, should we load a blank or spinner in the card?